resources/js/oe_whitelabel.project_status.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file
+ * Attaches behaviors for the project status element.
+ */
+(function (bootstrap, Drupal, $) {
+
+  const colorClasses = [
+    'bg-secondary',
+    'bg-info',
+    'bg-dark',
+  ];
+
+  /**
+   * Animates the project status badge and progress bar.
+   *
+   * @type {Drupal~behavior}
+   *
+   * @prop {Drupal~behaviorAttach} attach
+   *   Initialises the behavior.
+   */
+  Drupal.behaviors.projectStatus = {
+    attach: function (context) {
+      $('.bcl-project-status', context).once('bcl-project-status').each(function () {
+        const $element = $(this);
+        const msBegin = $element.data('start-timestamp') * 1000;
+        const msEnd = $element.data('end-timestamp') * 1000;
+        const statusLabels = $element.data('status-labels').split('|');
+        const msNow = Date.now();
+        const status = (msNow >= msBegin) + (msNow > msEnd);
+        const progress01 = Math.max(0, Math.min(1, (msNow - msBegin) / (msEnd - msBegin)));
+        const percent = Math.round(progress01 * 200) / 2;
```

Comment:
How come this 2 factor? Can't it be * 100 without the / 2? We need documentation to understand the reasons.

Reply:
I am now rounding to 1%.
But to explain my idea, my feeling was `37.5%` looks ok but `37.1%` already looks too fine-grained.
Now the new rationale is `37%` looks ok but `37.5%` looks too fine-grained.

These attributes are for accesibility, but I can only speculate how they may be perceived in practice.